src/test/constants.ts

```diff
@@ -0,0 +1,479 @@
+import { PullRequestPayload, PullRequest } from "../github";
```

Comment:
This file is getting rather long. 
Maybe we could split it up into smaller files?
Maybe we could reduce the sample data to those parts of the data structure which is relevant for us?

It looks like we are using real data here. Which is good. At the same time this copy of the data might deviate from the actual data when the API changes. If we limit these examples to the stuff we care about, we have a shorter file, it is clearer what we care about and chances are higher that these bits fit to the actual data.

But I have to admit that I am of two minds about this. I definitely see a benefit in using actual data in an unredacted form. However, maybe in this case we could write a script that regenerated this file by making and recording calls to GitHub backend,

Reply:
It actually is already redacted from the mentioned url (to make it fit the data types, but also to have some extra options available). We should probably just slim it down. 

I don't think we have to "generate" this file automatically. If Github's API changes, we just have a broken project, but our error reporting should be good enough to make it clear that it's not working anymore. As this project is not important enough to be fully resilient against API changes. Do you agree?

Reply:
Sounds good. Then maybe just slimming it down a little more

Reply:
@pihme I've tried to slim it down, but it's relatively difficult to maintain the type restrictions that we get from the currently used types. Perhaps we could try this some time together (pair programming), but for now I'd like to just leave this as it is and finish this PR. Do you agree?